### DISCUSSION TITLE
Align HAOS versioning to core and supervisor number standard [year].[month].[rev]?

### DISCUSSION THREAD

**Discussion by alvinchen1** [home-assistant/operating-system#1773] [Ideas]
Dumb Question.  Was wondering is there a reason HAOS doesn't share its version and numbering standard with Home Assistant core and supervisor?  With approximate one month releases, couldn't it have been:

- Release 7 build 4 = haos 2022.2.2
- Release 7 build 3 = haos 2022.2.1
- Release 7 build 2 = haos 2022.1.1
- Release 7 build 1 = haos 2021.12.2
- Release 7 build 0 = haos 2021.12.1

etc?  Just curious. Thanks for all the work!

**Comment by agners** [comment]
We discussed and considered this previously. One reason we stayed with the versioning scheme so far is to avoid confusion. People already regularly mix up the version numbers all the time already now. Having OS also using CalVer will cause even more mix-ups.

**Reply by alvinchen1** [reply]
Interesting!  Was just curious about that.  I'm sure smarter brains than mine have worked through issues like that.  I appreciate your time in responding.  

I asked cause I am helping two other very novice users get their HA setup.  One of them asked me about that, and I didn't have a good answer for it, and I thought, hmmm, not a bad question.
